src/xdocs/index.xml.vm

```diff
@@ -88,14 +88,16 @@
     </section>
 
     <section name="Development Cycle">
-      <p>
-        <b>Releases will happen at the end of each month</b>
-        if functional changes exist in the
-        <a href="https://github.com/checkstyle/checkstyle/commits/master">
-            master branch of our repo</a>.
-        For the exact day, please follow our
-        <a href="https://github.com/checkstyle/checkstyle/milestones">milestones</a>.
-      </p>
+      <b>Release:</b><br/>
+      <ul>
+        <li>happens at the end of each month if functional changes exist in the
+          <a href="https://github.com/checkstyle/checkstyle/commits/master">
+          master branch of our repo</a>
+        </li>
+        <li>can happen by request of author of merged Pull Request(s),
```

Comment:
This seems unnecessarily restrictive. What if I report a bug, but someone else fixes it?  Can I request a release?

Reply:
I am ok to release by any request, as it is single click, we can do this from phones at any time. If someone needs a fix to be released, let's do it.
I doubt it would happen frequently, as usually who do fix needs it, all others might not exposed to such problem.

Reply:
@rnveach , do you agree ?

`can happen by request of author of merged Pull any user who is affected and what to use fix quicker`

Reply:
````
can happen by request by any user who is impacted, but it is not always guaranteed.
````

Reply:
done.